README.md

```diff
@@ -139,5 +141,27 @@ var MailExampleApp = React.createClass({
 
 On Android, the `callback` will only be called if an `error` occurs. The `event` argument is unused!
 
+## Example 2
+Check if the user has an email account configured.
+
+```js
+var Mailer = require('NativeModules').RNMail;
+
+Mailer.canSend((error, canSend) => {
```

Comment:
seems like error in the callback is always null. Seems like outcome `canSend` is always true or false. maybe keep the apis `Mailer.canSend(canSend => {});`?